FoodPin/FoodPin/View/RestaurantDetailTwoColumnCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RestaurantDetailTwoColumnCell.swift
+//  FoodPin
+//
+//  Created by Andreea Sburlea on 28.04.2022.
+//
+
+import UIKit
+
+class RestaurantDetailTwoColumnCell: UITableViewCell {
+
+    @IBOutlet private var column1TitleLabel: UILabel! {
```

Comment:
Any UI elements and objects in general should have intuitive names. You can use something like "addressTitleLabel"